src/pages/topics.tsx

```diff
@@ -0,0 +1,226 @@
+import { GetStaticProps } from 'next';
+import { clientFactory } from '../utils/client-factory';
+import { getAllMetaInfo } from '../service/get-all-meta-info';
+import { getURLPath } from '../utils/format-url-path';
+import Hero from '../components/hero';
+import { Crumb } from '../components/breadcrumbs/types';
+import { GridLayout, TypographyScale } from '@mdb/flora';
+
+interface Cta {
+    text: string;
+    url: string;
+}
+
+interface Topic {
+    title: string;
+    icon: string;
+    href: string;
+}
+
+interface TopicsProps {
+    category: string;
+    tagName: string;
+    description: string;
+    slug: string;
+    ctas: Cta[];
+    topics: Topic[];
+}
+
+const getProductsByName = (productsArray: string[], topics: TopicsProps[]) => {
+    const L1Products = topics.filter(
+        ({ category, tagName }) =>
+            category === 'L1Product' &&
+            productsArray.some(productName => productName === tagName)
+    );
+    return L1Products;
+};
+
+const ULStyles = {
+    listStyleType: 'none',
+    padding: 0,
+    marginBottom: ['section20', 'section20', 'section20', 'section40'],
+    display: 'grid',
+    columnGap: 'inc70',
+    gridTemplateColumns: [
+        'repeat(2, 1fr)',
+        'repeat(2, 1fr)',
+        'repeat(4, 1fr)',
+        'repeat(4, 1fr)',
+        'repeat(6, 1fr)',
+    ],
+    li: {
+        paddingBottom: ['inc30', 'inc30', 'inc30', 'inc40'],
+        'a:hover': {
+            textDecoration: 'underline !important',
+        },
+    },
+};
+
+const Topic = ({ topics }: { topics: TopicsProps[] }) => {
+    const crumbs: Crumb[] = [
+        { text: 'MongoDB Developer Center', url: '/developer' },
+        { text: 'Developer Topics', url: '/developer/topics' },
+    ];
+
+    const getExpertiseLevels = topics.filter(
+        ({ category }) => category === 'ExpertiseLevel'
+    );
+    const getLanguages = topics?.filter(
+        ({ category }) => category === 'ProgrammingLanguage'
+    );
+    const getTechnologies = topics?.filter(
+        ({ category }) => category === 'Technology'
+    );
+
+    const L1productsArray = [
+        'Atlas',
+        'Compass',
+        'Connectors',
+        'MongoDB',
+        'Ops Manager',
+        'Realm (Mobile)',
+    ];
+
+    const [Atlas, Compass, Connectors, MongoDB, OpsManager, Realm] =
+        getProductsByName(L1productsArray, topics);
+
+    return (
+        <>
+            <Hero crumbs={crumbs} name="All Topics" />
+            <GridLayout
+                sx={{
+                    paddingLeft: [
+                        'inc40',
+                        'inc40',
+                        'inc50',
+                        'inc70',
+                        'inc70',
+                        0,
+                    ],
```

Comment:
Just a nit and doesn't really matter, but you can use `null` is these arrays to allow the value to carry over from the last one. For example `['inc40', 'inc40', 'inc50', 'inc70', 'inc70', 0]` could be `['inc40', null, 'inc50', 'inc70', null, 0]`. It's not too helpful in this use case, but for longer expressions it's kinda nice.